src/extras/helpers/EdgesHelper.js

```diff
@@ -1,10 +1,19 @@
 /**
  * @author WestLangley / http://github.com/WestLangley
+ * @param object THREE.Mesh whose geometry will be used
+ * @param hex line color
+ * @param thresholdAngle the minimim angle (in degrees),
+ * between the face normals of adjacent faces,
+ * that is required to render an edge. A value of 10 means
+ * an edge is only rendered if the angle is at least 10 degrees.
  */
 
-THREE.EdgesHelper = function ( object, hex ) {
+THREE.EdgesHelper = function ( object, hex, thresholdAngle ) {
 
 	var color = ( hex !== undefined ) ? hex : 0xffffff;
+	thresholdAngle = ( thresholdAngle !== undefined ) ? thresholdAngle : 0.1;
```

Comment:
`acos( 0.9999 ) = 0.810`, so to make the code backwards-compatible, the default threshold angle should be 0.810. Since that is a bit silly, I think 1 degree is reasonble.

	thresholdAngle = ( thresholdAngle !== undefined ) ? thresholdAngle : 1; // degrees